src/ansys/tools/report.py

```diff
@@ -147,13 +147,21 @@ def project_info(self):
                     env_info_lines.append(f"{key:<30} {value}")
                     n_var += 1
 
-        # Now, take into account the default env variables
-        for ansys_default in __ANSYS_VARS__:
-            for key, value in os.environ.items():
-                if (ansys_default in key) and (key not in self._ansys_vars):
-                    env_info_lines.append(f"{key:<30} {value}")
-                    n_var += 1
-
+        # Loop over all environment variables
+        for key, value in os.environ.items():
+            # Now, take into account the default env variables
+            for ansys_default in __ANSYS_VARS__:
```

Comment:
For explicitly saying that this is a var prefix:

```suggestion
            for ansys_var_prefix in __ANSYS_VARS__:
```